Bottom: da05277e1871d53689bbd5b4e7de65b3ef44b46a
Top:    5ca0abffda92cfb8bbb0ec206cc04b5e4d30fc22
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-13 02:26:42 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
